addons/project/project.py

```diff
@@ -162,9 +162,9 @@ def _get_favorite(self, cr, uid, ids, name, args, context=None):
 
     def _set_favorite(self, cr, uid, ids, field_name, field_value, args, context=None):
         if field_value:
-            self.write(cr, uid, ids, {'favorite_user_ids': [(4, uid)]}, context=context)
+            self.write(cr, SUPERUSER_ID, ids, {'favorite_user_ids': [(4, uid)]}, context=context)
         else:
-            self.write(cr, uid, ids, {'favorite_user_ids': [(3, uid)]}, context=context)
+            self.write(cr, SUPERUSER_ID, ids, {'favorite_user_ids': [(3, uid)]}, context=context)
```

Comment:
@mge-odoo 
this will not work. because 
project user has only read access.
and from toggle_favorite method we are going to update is_favorite with normal project user so before _set_favorite method we got access rights validation Error.

we need to write with superuser in toggel_favorite method.